src/App.js

```diff
@@ -29,6 +29,16 @@ function App() {
     console.log(event.target.lname.value);
     console.log(event.target.phone.value);
 
+    fetch('http://localhost:3001/', {
+      method: 'POST',
+      body: JSON.stringify({firstName: event.target.fname.value, lastName: event.target.lname.value, phone: event.target.phone.value}),
```

Comment:
nitpick: It would probably look better if you had these as multiline
```
body: JSON.stringify({
  firstName: event.taarget.fname.value,
  lastName: event.target.lname.value,
  // etc
})
```
(nitpick means it doesn't really matter if you take the suggestion oir not )